native/chat/chat-thread-background.react.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import * as React from 'react';
+import ChatThreadList from './chat-thread-list.react';
+import type { ChatTopTabsNavigationProp } from './chat.react';
+import { threadInBackgroundChatList } from 'lib/shared/thread-utils';
+import { type ThreadInfo, type RawThreadInfo } from 'lib/types/thread-types';
+
+type BackgroundChatThreadListProps = {
+  navigation: ChatTopTabsNavigationProp<'BackgroundChatThreadList'>,
```

Comment:
We should avoid introducing new types using the ambiguous `{ }` syntax. You should always prefer either `{| |}` exact or `{ ... }` inexact syntax. More context [here](https://medium.com/flow-type/on-the-roadmap-exact-objects-by-default-16b72933c5cf), but basically `{ }` currently means inexact but will soon mean exact.

Whenever possible exact syntax is preferred, but in this case because we aren't specifying the `route` prop we have to use inexact:

```js
type BackgroundChatThreadListProps = {
  navigation: ChatTopTabsNavigationProp<'BackgroundChatThreadList'>,
  ...
};
```